.travis.yml

```diff
@@ -39,20 +39,20 @@ env:
     - OS_CODE_NAME2=bionic
     - ARCH=amd64
     - ROS_BUILDFARM_PULL_REQUEST_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+os: linux
```

Comment:
Why should we specify the default explicitly? Just for readability?

Reply:
Yeah, and to squelch the extra info being generated for each build. I can drop it if you think it makes things worse somehow.

Reply:
It can stay - I was just making sure I understand it correctly.